### ISSUE TITLE
merge existing structs

### ISSUE THREAD

**Issue by psifertex** [Vector35/binaryninja-api#2594]
If several structs have been created throughout analysis and then later it's discovered they are the same, it should be possible to merge them and all references into one unified struct

**Comment by plafosse** [comment]
Likely this would just simply be the ability replace all instances of one structure with another.

**Comment by nshp** [comment]
This was from a conversation we had. The super nice thing would be merging (with some interactive conflict resolution if necessary) the member types/names of the two structs.

**Comment by psifertex** [comment]
Yeah, the code to do that merging lives in the collaboration plugin so it's definitely possible.
